src/Microsoft.Extensions.CodeGeneration.Core/project.json

```diff
@@ -9,6 +9,7 @@
     "Microsoft.Extensions.CodeGeneration.Templating": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
```

Comment:
This project depends on `Utils` project below. So adding it there should be enough?